### PR TITLE
added an option to execute ipython

### PR DESCRIPTION
**Pull request by arielrossanigo**
Added an option use ipython instead of python shell. 

The option is -i | --ipython. This option add ipython as a dependency.

Examples:
`$ fades -i -d requests`

`$ echo "import requests; print(requests.__version__)" > foo.py`
`$ fades -i -d requests foo.py -i`

In the last example, first *i* means "use IPython" while second *i* is a parameter to IPython shell indicating that we want to use interactive session.

### CODE REVIEW

File: fades/main.py [hunk 1]
```diff
@@ -82,6 +82,8 @@ def go(version, argv):
     parser.add_argument('-p', '--python', action='store',
                         help=("Specify the Python interpreter to use.\n"
                               " Default is: %s") % (sys.executable,))
+    parser.add_argument('-i', '--ipython', action='store_true',
+                        help="use IPython shell.")
```
Comment (facundobatista) [inline fades/main.py]: Please add -i/--ipython options and explanation both to README and man page.

File: fades/main.py [hunk 2]
```diff
@@ -119,13 +121,19 @@ def go(version, argv):
         l.warning("Overriding 'quiet' option ('verbose' also requested)")
 
     # parse file and get deps
+    if args.ipython:
+        ipython_dep = parsing.parse_manual(['ipython'])
```
Comment (facundobatista) [inline fades/main.py]: Please log in debug that you're adding the ipython dependency

File: fades/main.py [hunk 3]
```diff
@@ -119,13 +121,19 @@ def go(version, argv):
...
+    else:
+        ipython_dep = []
     indicated_deps = parsing.parse_srcfile(args.child_program)
     l.debug("Dependencies from source file: %s", indicated_deps)
     reqfile_deps = parsing.parse_reqfile(args.requirement)
     l.debug("Dependencies from requirements file: %s", reqfile_deps)
     manual_deps = parsing.parse_manual(args.dependency)
     l.debug("Dependencies from parameters: %s", manual_deps)
-    indicated_deps = _merge_deps(indicated_deps, reqfile_deps, manual_deps)
+    indicated_deps = _merge_deps(
```
Comment (facundobatista) [inline fades/main.py]: No need to have multiple lines here, as column limit is 99 in this project :)

**Comment by facundobatista** [comment]
Awesome branch, thanks!! I added some inline comments ↑

**Comment by arielrossanigo** [comment]
I've just add some commits in order to support your comments

**Comment by facundobatista** [comment]
Awesome, thank you very much!

**Pull request merged**